doc/sphinx/proofs/writing-proofs/rewriting.rst

```diff
@@ -92,6 +92,8 @@ Rewriting with Leibniz and setoid equality
 
      If :n:`at @occs_nums` is specified, rewriting is always done with
      :ref:`setoid rewriting <generalizedrewriting>`, even for Leibniz’s equality.
+     However, note that `rewrite ... at` and `setoid_rewrite` don't behave identically.
```

Comment:
So that it's clearer to users why we specify that this form uses setoid rewriting:

```suggestion
     If :n:`at @occs_nums` is specified, rewriting is always done with
     :ref:`setoid rewriting <generalizedrewriting>`, even for Leibniz’s equality,
     which means that you must `Require Setoid` to use that form.
     However, note that :tacn:`rewrite` (even when using setoid rewriting) and
     :tacn:`setoid_rewrite` don't behave identically (as already mentioned above).
```